File: specification/specs/search/scroll/scroll/ScrollRequest.ts
@rest_spec_name('scroll')
class ScrollRequest extends RequestBase {
  path_parts?: {
    scroll_id?: Id
  }
  query_parameters?: {
    total_hits_as_integer?: boolean
    scroll?: Time
  }
  body?: {
    scroll?: Time
    scroll_id?: string
  }
}

File: output/csharp/Search/Scroll/Scroll/ScrollRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Search {

	public class ScrollRequest : RequestBase {
		
		[DataMember(Name="total_hits_as_integer")]
		public bool TotalHitsAsInteger { get; set; }


		[DataMember(Name="scroll")]
		public Time Scroll { get; set; }


		[DataMember(Name="scroll_id")]
		public string ScrollId { get; set; }

	}
}
